plugins/org.locationtech.udig.core.tests/src/org/locationtech/udig/core/NumberAdaptableFactory.java

```diff
@@ -0,0 +1,52 @@
+package org.locationtech.udig.core;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.sql.Date;

```

Comment:
Wondering why java.sql imports are necessary for this class, please re-organize imports. Any chance to provide Unit tests as well?

Reply:
A number of unit tests (not exhaustive) added to verify the possibility of adaptation during Feature copy.
Unnecessary imports removed.